### DISCUSSION TITLE
Adding in a 'No data' message to charts

### DISCUSSION THREAD

**Discussion by adamjdavies** [tremorlabs/tremor-npm#403] [Q&A]
Hi all, 

Firstly, big props to the Tremor team on what they've created. I've been able to whip up a nice little dashboard in under an hour. 

One question I did have was if there is an easy way to add a message like 'No data' to a chart? The data powering my dashboard is based on a specific time interval and there may be times where no data exists for that time period. I would love to be able to add in message directly on the chart saying 'no data available' or something similar. 

I can obviously go in and create some conditional UI to show this myself, but just wondered if there was an easier way to do it directly in the chart component. 

Thanks!

**Comment by mitrotasios** [comment]
Hey @adamjdavies sorry for the late response. That makes sense, would you like to submit an issue for this so that we can prioritise it better?

**Reply by adamjdavies** [reply]
Sure, I'll do that!
